tutorials.md.in

```diff
@@ -10,6 +10,7 @@ Ignition @IGN_DESIGNATION_CAP@ library and how to use the library effectively.
 1. \subpage introduction "Introduction"
 2. \subpage installation "Installation"
 3. \subpage physicsplugin "Understanding the Physics Plugin"
+3. \subpage switchphysicsengines "Switching physics engines"
```

Comment:
```suggestion
4. \subpage switchphysicsengines "Switching physics engines"
```
This is how it shows up without this fix (FYI, if you want to see the rendered version, build the code and open `doxygen/html/index.html` form the build directory):

![image](https://user-images.githubusercontent.com/206116/94941274-43914280-049a-11eb-84cc-b4c0e61c293f.png)